### PR TITLE
Dockerfile

### PR DESCRIPTION
**Pull request by slajax**
Add dockerfile

### CODE REVIEW

**Comment by slajax** [comment]
@lvivier @RongxinZhang please review, I need this merged to proceed.

**Comment by lvivier** [comment]
:+1:

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+start-app:
+	@./bin/logd --logentries ${LOGENTRIES_KEY} --segment ${SEGMENT_IO_SECRET}
```
Comment (RongxinZhang) [inline Makefile]: @slajax this should be: LOGENTRIES_APP_TOKEN  https://github.com/Retsly/log/blob/master/bin/logd#L50.

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,12 @@
+start:
+	@./bin/logd --logentries ${LOGENTRIES_KEY} --segment ${SEGMENT_IO_SECRET}
```
Comment (RongxinZhang) [inline Makefile]: This should be LOGENTRIES_APP_TOKEN. https://github.com/Retsly/log/blob/master/bin/logd#L50

**Comment by RongxinZhang** [comment]
@slajax looks good otherwise :+1:

File: Makefile (hunk 2 above)
Comment (slajax) [inline Makefile]: @RongxinZhang I forgot to mention, I've updated this in my docker branches.

**Comment by slajax** [comment]
ref Retsly/deploy#25 Retsly/deploy#27

**Comment by slajax** [comment]
ref Retsly/api#1016

**Pull request merged**